content/ytt/docs/latest/ytt-schema.md

```diff
@@ -0,0 +1,60 @@
+---
+title: ytt @schema/match
+---
+
+## Overview
+
+`ytt` schemas are currently in the **experimental** phase. To use schema features, include `--enable-experiment-schema`.
+
+Configuration authors use Schema to verify data value YAML structures and their values and define default data value configuration.
+
+Using a schema guarantees to templates that all data values exist and are of the correct type. This alleviates templates from doing existence and type checks themselves.
+
+Common use cases:
+
+1. Set default values
+1. Provide visibility to valid values that could be updated by the consumer 
+1. Provide visibility to valid types and YAML structure that could be updated by the consumer
```

Comment:
With my "new user" empathy cap on, I wonder if we've already motivated the core use case.

```suggestion
```